### PR TITLE
Surface Associated Pull Request Information

### PR DESCRIPTION
**Pull request by chidozieononiwu**
- Show Associated Pull Request Information

### CODE REVIEW

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html [hunk 1]
```diff
@@ -51,6 +51,22 @@
     </ul>
 </app-page-options-section>
 
+<app-page-options-section *ngIf="associatedPullRequests.length > 0" sectionName="Associated Pull Requests">
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:54]: Will there be more than one pull request for a revision? A PR based revision is always associated with only one pull request. right?

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html [hunk 2]
```diff
@@ -51,6 +51,22 @@
...
+    <ul class="list-group">
+        <li class="list-group-item" *ngFor="let pullRequest of associatedPullRequests">
+            <a href="https://github.com/{{pullRequest.repoName}}/pull/{{pullRequest.pullRequestNumber}}" target="_blank">{{pullRequest.repoName}}/{{pullRequest.pullRequestNumber}}</a>
+        </li>
+    </ul>
+</app-page-options-section>
+
+<app-page-options-section *ngIf="pullRequestsOfAssociatedAPIRevisions.length > 0" sectionName="Associated APIRevisions" [collapsedInput]="true">
```
Comment (praveenkuttappan) [inline src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:62]: `Associated APIRevisions` is the section name.  APIRevisions is a DB name and I don't think that's the label we should use on UI if label and section name are same. We can just say `Associated Revisions`

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html (hunk 1 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:54]: Yes that is very much the case

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html (hunk 2 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:62]: I think APIRevisoin is fine to differentiate from Sample revision, by the way that's the current name in the old UI.

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:54]: Can you send me an example where a revision contains more than one PR? That should not be the case. A pull request revision should associate with only one PR.

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html (hunk 2 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:62]: `Associated API Revisions`

Comment (praveenkuttappan) [inline src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:62]:
```suggestion
<app-page-options-section *ngIf="pullRequestsOfAssociatedAPIRevisions.length > 0" sectionName="Associated API Revisions" [collapsedInput]="true">
```

File: src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html (hunk 1 above)
Comment (chidozieononiwu) [reply src/dotnet/APIView/ClientSPA/src/app/_components/review-page-options/review-page-options.component.html:54]: Actually, I was agreeing with your previous assertion. the PR revision is created via one PR. This is just a check to make sure that the option is not shown when there is no associatedPullRequests

**Review by maririos — APPROVED** [review]

**Pull request merged**